hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyService.java

```diff
@@ -716,8 +729,8 @@ protected RequestInterceptor createRequestInterceptorChain() {
         }
       } catch (ClassNotFoundException e) {
         throw new YarnRuntimeException(
-            "Could not instantiate ApplicationMasterRequestInterceptor: "
-                + interceptorClassName, e);
+            "Could not instantiate ApplicationMasterRequestInterceptor: " +
+                    interceptorClassName, e);
```

Comment:
This indentation should be different right?

Reply:
I will fix it.